README.md

```diff
@@ -29,9 +29,9 @@ Initially this module was named Net::LDNS.
 ## Dependencies and compatibility
 
 Run-time dependencies:
- * `openssl`
+ * `openssl` (openssl >= 1.1.0 unless [Ed25519] is disabled)
```

Comment:
Since Ed25519 is disabled by default in OpenSSL 1.1.0 (but enabled by default in 1.1.1) I think the text is misleading. A standard package of 1.1.0 will probably not be good enough.

Reply:
I guess that depends on the whims and policies of whoever compiles the source. Ed25519 could be enabled for some openssl = 1.1.0 and disabled for some openssl = 1.1.1. What we can say for sure is that it won't work for openssl < 1.1.0 if Ed25519 is enabled in Zonemaster LDNS.

Reply:
So
```
(openssl >= 1.1.0 with support for Ed25519 unless [Ed25519] is disabled)
```

I do not want people to upgrade to a "standard" OpenSSL 1.1.0 package thinking that it will solve the problem.